data/blog/app-feedback-program.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Electron App Feedback Program
+author: sofianguy
+date: '2018-10-04'
+---
+
+Electron is working on building a streamlined release process and having faster releases. To help with that, we started the App Feedback Program for large-scale Electron apps to test our beta releases and report app-specific issues to the Electron team. We use this program to help us prioritize work and get applications upgraded to the next stable release as soon as possible.
+
+## Who can join?
+Our criteria and expectations for apps joining this program include the following items:
+- Test your app during the beta period for 10,000+ user-hours.
+- Have a single point-person to speak for the app's Electron bugs, app blockers, and check-in weekly, either during our regularly scheduled online meeting, or asynchronously with @sofianguy
+- You are willing to share the following information listed in the next question.
+
+## What info does my Electron app have to share?
+- Total user-hours your app has been running with any beta release.
+- Version of Electron that your app is testing with (e.g., 3.0.0-beta.3)
+- Any bugs preventing your application from upgrading to the release line being beta tested
+
+## User-hours
+We expect participants to test at least 10,000 user-hours as a minimum threshold for Electron to feel more comfortable releasing stable versions.
+- 10 user-hours could be 10 people testing for one hour, or one person testing for 10 hours.
+- You can split the testing between beta releases, for example test for 5,000 user-hours on 3.0.0-beta.2 and then test for 5,000 user-hours on 3.0.0-beta.5. More is better, but we understand that some applications cannot test every beta release.
+- CI or QA hours do not count towards the total, however internal releases do count.
+
+## Why should my Electron app join?
+Your app's bugs will be prioritized on the core Electron team's radar. In return, this is helpful for Electron to see how it is doing and what work needs to be prioritized.
+
+## Electron prioritizing workflow
+Starting in the Electron version 4.0.x cycle, we are going to focus on helping the app that is closest to shipping to the next largest group of users.
+For example, if company X has an Electron bug blocking them from releasing to 1,000 more users and company Y has a blocker from releasing to 500 more users, we will prioritize company X first and then help company Y next.
```

Comment:
This might demotivate some smaller potential partners. Is that intentional? If not, it might be better to omit this section.

Reply:
We should just say that joining is not a guarantee all bugs will be fixed, but it definitely helps us identify priorities. We also make a significant effort to get apps giving us lots of feedback across the finish line.

Reply:
Really not a fan of the "lifeboat game" framing, either in practice or in this blog post.

Talking about it in terms of identifying priorities works better. I like the positive feedback loop of going the extra mile for projects that went the extra mile for us.

Reply:
I will remote this section for the blog post and instead I'll tell participants that not every bug will be fixed.